Makefile

```diff
@@ -123,6 +123,11 @@ verify:
 	@echo "===> Verifying documentation formatting for website <==="
 	$(CURDIR)/hack/verify-docs-for-website.sh
 
+.PHONY: toc
+toc:
+	@echo "===> Generating Table of Contents for Antrea docs <==="
+	GO=$(GO) $(CURDIR)/hack/update-toc.sh
```

Comment:
is this addition related to helm charts?

Reply:
No. Migrating this to make toc generation more convenient. Put it in a separate PR https://github.com/antrea-io/theia/pull/21 Thanks!